import-zone

```diff
@@ -119,6 +119,7 @@ unless ($pzoneid > 0) {
   }
   $zonehash{txt}=[];
   foreach $rtmp (@{$rec->{TXT}}) { push @{$zonehash{txt}}, [0,$rtmp,'']; }
+  foreach $rtmp (@{$rec->{CAA}}) { push @{$zonehash{zentries}}, [0,"IN CAA $rtmp",'']; }
```

Comment:
The record here should be "@ IN CAA $rtmp", because this is supposed to be RR on the zone.
(otherwise it would end up on the last host)